spec/features/peep_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'capybara'
+
+feature 'allpeeps' do
+  scenario 'view all peeps' do
+    connection = PG.connect(dbname: 'chitter_manager_test')
```

Comment:
This could be worth wrapping in a `before` block to make it clear that this is part of setting up the tests